src/status_im/ui/screens/keycard/views.cljs

```diff
@@ -0,0 +1,114 @@
+(ns status-im.ui.screens.keycard.views
```

Comment:
could you move long styles maps to styles ns

Reply:
@flexsurfer How many lines considered long? I used to move all styles to styles ns before, but stopped doing that as you need to come up with a name for every style you move so i find inlining styles more convenient unless you duplicate them.

Reply:
do you feel comfortable to read this code?

Reply:
@flexsurfer yes. Can you provide example of long maps you find unreadable?

For example, long maps vs long maps moved to styles:

```cljs
(defn connection-lost []
  [react/view {:flex             1
               :justify-content  :center
               :align-items      :center
               :background-color colors/gray-transparent-40}
   [react/view {:background-color colors/white
                :height           478
                :width            "85%"
                :border-radius    16
                :flex-direction   :column
                :justify-content  :space-between
                :align-items      :center}
    [react/view {:margin-top 32}
     [react/text {:style {:typography :title-bold
                          :text-align :center}}
      (i18n/label :t/connection-with-the-card-lost)]
     [react/view {:margin-top 16}
      [react/text {:style {:color      colors/gray
                           :text-align :center}}
       (i18n/label :t/connection-with-the-card-lost-text)]]]
    [react/view {:margin-top 16}
     [react/image {:source      (resources/get-image :keycard-connection)
                   :resize-mode :center
                   :style       {:width  200
                                 :height 200}}]]
    [react/view {:margin-bottom 43}
     [react/touchable-highlight
      {:on-press #(re-frame/dispatch [:keycard.onboarding.connection-lost.ui/cancel-setup-pressed])}
      [react/text {:style {:color      colors/red
                           :text-align :center}}
       (i18n/label :t/cancel-keycard-setup)]]]]])
```

vs

```cljs
(defn connection-lost []
  [react/view styles/container
   [react/view styles/inner-container
    [react/view {:margin-top 32}
     [react/text {:style {:typography :title-bold
                          :text-align :center}}
      (i18n/label :t/connection-with-the-card-lost)]
     [react/view {:margin-top 16}
      [react/text {:style {:color      colors/gray
                           :text-align :center}}
       (i18n/label :t/connection-with-the-card-lost-text)]]]
    [react/view {:margin-top 16}
     [react/image {:source      (resources/get-image :keycard-connection)
                   :resize-mode :center
                   :style       {:width  200
                                 :height 200}}]]
    [react/view {:margin-bottom 43}
     [react/touchable-highlight
      {:on-press #(re-frame/dispatch [:keycard.onboarding.connection-lost.ui/cancel-setup-pressed])}
      [react/text {:style {:color      colors/red
                           :text-align :center}}
       (i18n/label :t/cancel-keycard-setup)]]]]])
```